web/book/src/SUMMARY.md

```diff
@@ -67,7 +74,6 @@
 
 - [Integrations](./integrations/README.md)
 
-  - [CLI](./integrations/cli.md)
```

Comment:
I agree this doesn't really fit at the moment.

But I think we need some list of "How do I use PRQL?", which this is very much part of. To some extent we get that from the https://prql-lang.org/faq/, but having a list here would be good too. The proposed change better reflects how we produce these but arguably is worse on how they're consumed.

One idea would be changing this section to "Tools & Integrations", retaining this as a link, even if the page here just points to the https://github.com/PRQL/prql/pull/2648/files#diff-5209855e27f70bffaa6997cd87e9aed958ae9c4bb768f841bfff586b0c55562fR1.

WDYT?